INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:367] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192210]: [success] (in 84 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:369] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192211]: [success] (in 87 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:371] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192212]: [success] (in 85 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:373] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192213]: [success] (in 89 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:375] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192214]: [success] (in 76 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:377] in chat [private:880629533]
ERROR:asyncio:Task exception was never retrieved
future: <Task finished name='Task-15' coro=<Dispatcher._process_polling_updates() done, defined at /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=ProgrammingError('Incorrect number of bindings supplied. The current statement uses 0, and there are 2 supplied.')>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Users/igormihajlec/Bot/bot.py", line 59, in send_timetable
    l = database.get_id(las, en)
  File "/Users/igormihajlec/Bot/sqlighter.py", line 54, in get_id
    result = self.cursor.execute("SELECT id, value_teachers FROM TEACHERS WHERE last_name = '?' AND initials = '?'", (last_name, initials)).fetchall()
sqlite3.ProgrammingError: Incorrect number of bindings supplied. The current statement uses 0, and there are 2 supplied.
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:378] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192216]: [success] (in 125 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:380] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192217]: [success] (in 140 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:382] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192218]: [success] (in 104 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:384] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192219]: [success] (in 90 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:386] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192220]: [success] (in 144 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:388] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192221]: [success] (in 158 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:390] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192222]: [success] (in 190 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:392] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192223]: [success] (in 112 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:394] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192224]: [success] (in 108 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:396] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192225]: [success] (in 70 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:398] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192226]: [success] (in 167 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:400] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192227]: [success] (in 93 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:402] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192228]: [success] (in 80 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:404] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192229]: [success] (in 100 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:406] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192230]: [success] (in 72 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:408] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192231]: [success] (in 94 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:410] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192232]: [success] (in 115 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:412] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192233]: [success] (in 102 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:414] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192234]: [success] (in 96 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:416] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192235]: [success] (in 67 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:418] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192236]: [success] (in 115 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:420] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192237]: [success] (in 95 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:422] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192238]: [success] (in 84 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:424] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192239]: [success] (in 134 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:426] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192240]: [success] (in 155 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:428] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192241]: [success] (in 103 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:430] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192242]: [success] (in 120 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:432] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192243]: [success] (in 97 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:434] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192244]: [success] (in 90 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:436] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192245]: [success] (in 103 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:438] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192246]: [success] (in 125 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:440] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192247]: [success] (in 77 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:442] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192248]: [success] (in 98 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:444] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192249]: [success] (in 108 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:446] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192250]: [success] (in 95 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:448] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192251]: [success] (in 147 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:450] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192252]: [failed] (in 1 ms)
ERROR:asyncio:Task exception was never retrieved
future: <Task finished name='Task-20' coro=<Dispatcher._process_polling_updates() done, defined at /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=OperationalError('near "TEACHERS": syntax error')>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Users/igormihajlec/Bot/bot.py", line 60, in send_timetable
    l = database.get_id(las, en)
  File "/Users/igormihajlec/Bot/sqlighter.py", line 54, in get_id
    result = self.cursor.execute("SELECT id, value_teachers last_nameFROM TEACHERS WHERE last_name = ? AND initials = ?", (last_name, initials)).fetchall()
sqlite3.OperationalError: near "TEACHERS": syntax error
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:451] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192253]: [success] (in 173 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:453] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192254]: [success] (in 93 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:455] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192255]: [failed] (in 2 ms)
ERROR:asyncio:Task exception was never retrieved
future: <Task finished name='Task-16' coro=<Dispatcher._process_polling_updates() done, defined at /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=ValueError('not enough values to unpack (expected 2, got 1)')>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Users/igormihajlec/Bot/bot.py", line 64, in send_timetable
    id, value = l
ValueError: not enough values to unpack (expected 2, got 1)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:456] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192256]: [success] (in 94 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:458] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192257]: [success] (in 147 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:460] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192258]: [failed] (in 1 ms)
ERROR:asyncio:Task exception was never retrieved
future: <Task finished name='Task-16' coro=<Dispatcher._process_polling_updates() done, defined at /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=UnboundLocalError("local variable 's_tt_1' referenced before assignment")>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Users/igormihajlec/Bot/bot.py", line 88, in send_timetable
    s_tt_1 += f"{p}{item[4]} {item[5]} {item[6]} {item[7]} {item[8]}\n"
UnboundLocalError: local variable 's_tt_1' referenced before assignment
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:461] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192259]: [success] (in 90 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:463] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192260]: [success] (in 98 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:465] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192261]: [failed] (in 1 ms)
ERROR:asyncio:Task exception was never retrieved
future: <Task finished name='Task-15' coro=<Dispatcher._process_polling_updates() done, defined at /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=AttributeError("'int' object has no attribute 'isdigit'")>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Users/igormihajlec/Bot/bot.py", line 87, in send_timetable
    p = "" if item[3].isdigit() else item[3]
AttributeError: 'int' object has no attribute 'isdigit'
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:466] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192262]: [success] (in 103 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:468] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192263]: [success] (in 151 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:470] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192264]: [failed] (in 2 ms)
ERROR:asyncio:Task exception was never retrieved
future: <Task finished name='Task-16' coro=<Dispatcher._process_polling_updates() done, defined at /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=AttributeError("'int' object has no attribute 'isdigit'")>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Users/igormihajlec/Bot/bot.py", line 83, in send_timetable
    p = "" if item[3].isdigit() else item[3]
AttributeError: 'int' object has no attribute 'isdigit'
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:471] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192265]: [success] (in 100 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:473] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192266]: [success] (in 87 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:475] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192267]: [success] (in 97 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:477] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192268]: [success] (in 80 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:479] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192269]: [success] (in 76 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:481] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192270]: [success] (in 130 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:483] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192271]: [success] (in 101 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:485] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192272]: [success] (in 99 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:487] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192273]: [success] (in 105 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:489] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192274]: [success] (in 71 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:491] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192275]: [success] (in 168 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:493] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192276]: [success] (in 108 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:495] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192277]: [success] (in 90 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:497] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192278]: [success] (in 95 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:499] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192279]: [success] (in 72 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:501] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192280]: [success] (in 81 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:503] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192281]: [success] (in 143 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:505] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192282]: [success] (in 120 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:507] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192283]: [success] (in 104 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:509] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192284]: [success] (in 94 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:511] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192285]: [success] (in 109 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:513] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192286]: [success] (in 85 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:515] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192287]: [success] (in 71 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:517] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192288]: [success] (in 82 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:519] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192289]: [success] (in 178 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:521] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192290]: [success] (in 85 ms)
INFO:aiogram.contrib.middlewares.logging:Received message [ID:523] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192291]: [success] (in 194 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:525] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192292]: [failed] (in 1 ms)
ERROR:asyncio:Task exception was never retrieved
future: <Task finished name='Task-7' coro=<Dispatcher._process_polling_updates() done, defined at /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=AttributeError("'Bot' object has no attribute 'send_mesaage'")>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "/Users/igormihajlec/Bot/bot.py", line 29, in start_command
    await bot.send_mesaage(880629533,"Я готов любить и быть с тобой максимально долго, на сколько это возможно❤️‍🩹")
AttributeError: 'Bot' object has no attribute 'send_mesaage'
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:526] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192293]: [success] (in 176 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
INFO:aiogram:Bot: exemple_guap_bot [@exemple_guap_bot]
INFO:aiogram.dispatcher.dispatcher:Start polling.
INFO:aiogram.contrib.middlewares.logging:Received message [ID:529] in chat [private:880629533]
INFO:aiogram.contrib.middlewares.logging:Process update [ID:8192294]: [success] (in 331 ms)
INFO:aiogram.dispatcher.dispatcher:Stop polling...
WARNING:aiogram:Goodbye!
